plugins/helo.checks.js

```diff
@@ -402,7 +402,7 @@ exports.forward_dns = function (next, connection, helo) {
         return next();
     }
 
-    const cb = function (err, ips) {
+    const cb = (err, ips) => {
```

Comment:
```suggestion
    function cb (err, ips) {
```

Reply:
this actually causes another linter error by changing this to a function